test/e2e/plugins/server-info.go

```diff
@@ -32,7 +32,7 @@ var _ = ginkgo.Describe("server-info plugin", func() {
 	}
 	s := scaffold.NewScaffold(opts)
 
-	ginkgo.It("enable server-info plugin", func() {
+	ginkgo.It("enable server-info plugin_", func() {
```

Comment:
@tao12345666333 
I have run into an unexpected problem.
I just have modified the e2e test file for "global_rule". 
But it transferred the CI error reporting to the "server-info" plugin's e2e test file.

I have just tried as follows.
1. reinstall docker containers after run 
`sudo docker rm $(docker ps -a -q)` and 
`sudo rmi $(docker images -a)` and 
`sudo docker volume prune`.
2. and `git reset` to the version that we did not change the apisix-alpine(version is reseted to 2.12). There still be problem. 
3. The error message is `expect is True, actual is false`. (Sry, forgive me have delete all data and do not copy the former error message.

I am looking forward to hear from you!

Reply:
@Chever-John you can just ignore this or disable server-info plugin test case.

In APISIX v2.13 this plugin is disabled by default.

Reply:
clear！